plugins/doc_fragments/aci.py

```diff
@@ -94,6 +95,13 @@ class ModuleDocFragment(object):
     - Path to a file that will be used to dump the ACI JSON configuration objects generated by the module.
     - If the value is not specified in the task, the value of environment variable C(ACI_OUTPUT_PATH) will be used instead.
     type: str
+  no_verification:
```

Comment:
Is "no_verification" the final name?  
I believe the second GET request was implemented for informational purposes/verification.
How about "suppress_details"?

Reply:
I feel like `suppress_details` might be a little daunting for users 😄. 
How do you feel about `suppress_verification`? Does that sound good? 
I will also add some aliases.

Reply:
Fine by me